ui/desktop/app/templates/scopes/scope/projects/settings/index.hbs

```diff
@@ -3,4 +3,61 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-{{outlet}}
\ No newline at end of file
+<SettingsCard @header={{t 'titles.application'}} @icon='boundary'>
+  <div class='content'>
+    <div class='section'>
+      <div>
+        <Hds::Text::Display @tag='h5'>
+          {{t 'titles.version'}}
+        </Hds::Text::Display>
+        <Hds::Text::Body @tag='p' @size='300'>
+          {{@model.versionNumber}}
+        </Hds::Text::Body>
+      </div>
+      <div>
+        <Hds::Text::Display @tag='h5'>
+          {{t 'titles.cli'}}
+        </Hds::Text::Display>
+        <Hds::Text::Body @tag='p' @size='300'>
+          {{#if @model.cliExists}}
+            {{t 'states.installed'}}
+          {{else}}
+            {{t 'states.not-available'}}
+          {{/if}}
```

Comment:
I don't think this is very useful, since it should never be the case where the CLI isn't installed. We can show the `not-available` message if for some reason it isn't since that should be an exceptional case, but shouldn't we display the CLI version here? And the Version on line 14 be the desktop client version?

Reply:
Indeed, I just randomly tested (for something else) last week to run the DC with no CLI and it's useless and it breaks, so I do strongly believe no one will be able to open DC, connect to a controller and show settings page unless there is a CLI present on the system.

Reply:
removed this